pkg/operator/ceph/csi/spec.go

```diff
@@ -521,9 +526,10 @@ func deleteCSIDriverResources(
 	return succeeded
 }
 
-func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, rookclientset rookclient.Interface) (bool, error) {
+func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, ctx *clusterd.Context) (bool, error) {
```

Comment:
The context must be the first element in the function's signature.

Reply:
updated